src/identityApiClient.interfaces.ts

```diff
@@ -0,0 +1,44 @@
+import { IdentityApiData, MPID, UserIdentities } from '@mparticle/web-sdk';
+import {
+    IdentityCallback,
+    IIdentityResponse,
+} from './identity-user-interfaces';
+import {
+    IAliasRequest,
+    IAliasCallback,
+    IIdentityRequest,
+    IdentityAPIMethod,
+    IIdentity,
+} from './identity.interfaces';
+import { MParticleWebSDK } from './sdkRuntimeModels';
+
+export interface IIdentityApiClient {
```

Comment:
I actually find interfaces that are not yet being used to be easier to debug when they _are_ used.  I think a better approach for this would be to PR this into an IdentityAPI refactor feature branch, then do https://mparticle-eng.atlassian.net/browse/SQDSDKS-6679 into the same one.  This way we are more certain these types are correct based on implementation/linting int he IDE, as opposed to thinking it _looks_ fine.  Let me know if that is clear.

Reply:
I normally agree with you but one of the biggest struggles with this migration is the discrepancy between what we _think_ the interfaces should be and what _actually_ is happening in the code.

In the past, when trying to do the migration in one shot, it proved difficult to keep the migration in scope because of this discrepancy. I think the better approach would be to define the intended interfaces, and then update those interfaces if we realized we missed something or plan out a future revision to keep things in scope.